harness/determined/cli/workspace.py

```diff
@@ -385,6 +511,46 @@ def yaml_file_arg(val: str) -> Any:
                     *CHECKPOINT_STORAGE_WORKSPACE_ARGS,
                     *DEFAULT_POOL_ARGS,
                     cli.Arg("--json", action="store_true", help="print as JSON"),
+                    cli.Arg(
+                        "--cluster-name",
+                        type=str,
+                        help="cluster within which we create the workspace-namespace binding. This \
+                        argument is optional for single Kubernetes RM, but is required when \
+                            using multiple resource managers",
```

Comment:
Is there a way we can explain `--cluster-name` clearer? We may not want to say --cluster-name is required when using multiple resource managers since we support `det w create myws --auto-create-namespace-all-clusters`?
```       
 raise api.errors.BadRequestException(
             "must provide --namespace NAMESPACE or --auto-create-namespace, or remove "
             + "--cluster-name CLUSTER_NAME and specify --auto-create-namespace-all-clusters"
         )
```